.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +0,0 @@
-<!--
```

Comment:
What was this file for?

Reply:
I don't know, but I decided to just go ahead and get rid of it.